src/_selectors/AssetIndexSelectors.js

```diff
@@ -23,7 +24,32 @@ export const shownAssetIndexRowsSelector = createSelector(
     }
 );
 
-export const assetIndexHeadersSelector = createSelector(
+const assetIndexRowToDuration = row =>
+    row ? `${row[2]}–${row[3]}` : '—';
+
+const assetIndexTableSelector = createSelector(
+    shownAssetIndexRowsSelector,
```

Comment:
One simple improvement we can do is to store the proper structure in redux-store instead of transforming it afterwards ~

Right now I converted them into a tabular form, ie. 2d arrays
I've check and seems only AssetIndex component is depending on this data, so it's probably save to change redux store

The structure is 
```javascript
[
  ['Up/Down', 'Ends In/Out', ...],
  ['Random 100', '5t-10t', ....],
  ['Random 50', '5t-10t', ....],
  ....
]
```

Do you think it's ok to store this 2d arrays in redux store instead?